tree/tree/src/TChain.cxx

```diff
@@ -2692,11 +2707,53 @@ void TChain::SetEntryList(TEntryList *elist, Option_t *opt)
    TString treename, filename;
 
    TEntryList *templist = 0;
+
+   const auto *subentrylists = elist->GetLists();
+   if(strcmp(opt, "sync") == 0){
```

Comment:
does that mean i can't pass both `"sync"` and `"ne"`?

Reply:
Mmmm yes currently it's like that. "sync" and "ne" work on two different aspects (supposing chain and entrylist have the same sub files/lists vs avoiding retrieving full paths to files), so I even if one wrote them together the "sync" option would always take precedence and route the SetEntryList function to its own route.

What do you think? Should I add in the document that the two options don't mix with each other? Or something else?

Reply:
afaict the two options are orthogonal so there is no reason they can't both be specified?

Reply:
I understand your point but I think even if one specified "sync ne" the sync option would branch out and call
`templist = static_cast<TEntryList*>(subentrylists->At(ie))`, so the "ne" option would be completely ignored anyway because it should be passed to the function `TEntryList::GetEntryList` which is called only if the sync option is not active (line 2762)

Reply:
ok, got it